cmd/satellite/main.go

```diff
@@ -95,6 +102,11 @@ var (
 		Database string `help:"satellite database connection string" releaseDefault:"postgres://" devDefault:"postgres://"`
 		Output   string `help:"destination of report output" default:""`
 	}
+	gracefulExitCfg struct {
+		Database  string `help:"satellite database connection string" releaseDefault:"postgres://" devDefault:"sqlite3://$CONFDIR/master.db"`
```

Comment:
Why have --database if it is still required to have have the satellite config directory. Is there a case where you would need to point to a different DB than what is configured in config.yaml?

Reply:
Probably not - I was mainly following the pattern of the other `reports` command

Reply:
Actually, the way we access the database in `generateGracefulExitCSV` is like this:
`db, err := satellitedb.New(zap.L().Named("db"), gracefulExitCfg.Database)`
We don't actually load anything from the config file for this command it seems. I think we should leave the database arg here.

Reply:
I think the database string from the satellite config.yaml is loaded into `gracefulExitCfg.Database` anyway